recipes/iqtree/build.sh

```diff
@@ -1,11 +1,18 @@
 #!/bin/bash
-export CFLAGS="-I$PREFIX/include"
-export LDFLAGS="-L$PREFIX/lib"
+export CFLAGS="$CFLAGS -I$PREFIX/include"
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib"
 export CPATH=${PREFIX}/include
 
+if [ "$(uname)" == Darwin ]; then
+    export MACOSX_DEPLOYMENT_TARGET=10.7
+    export CMAKE_C_COMPILER="clang"
+    export CMAKE_CXX_COMPILER="clang++"
```

Comment:
Can you toss this env var in here, too?

```bash
export CMAKE_TRY_COMPILE_TARGET_TYPE=STATIC_LIBRARY
```

This changes the way cmake performs its pre-run checks.